plugins/pipeline_plugin/operators/RCloneOperator.py

```diff
@@ -0,0 +1,48 @@
+import os
+import tempfile
+from airflow.utils.decorators import apply_defaults
+from pipeline_plugin.config import config
+from pipeline_plugin.utils.google_cloud_storage import download_file
+from pathlib import Path
+
+from pipeline_plugin.operators.BaseMapActionOperator import MapActionOperator
+import subprocess
+
+
+def sync_from_gcp_to_gdrive():
+    gcp_path = config.get_data_bucket_name()
+    gdrive_folder_id = config.get_google_drive_output_folder_id()
+
+    _, service_account_auth_path = tempfile.mkstemp(suffix='.json')
+    gcsc = GoogleCloudStorageClient()
```

Comment:
I think `GoogleCloudStorageClient` needs to be imported

Reply:
Done